### DISCUSSION TITLE
Add Debian repository to supported package registries

### DISCUSSION THREAD

**Discussion by silopolis** [community/community#39006] [Actions]
### Select Topic Area

General

### Body

Hi,

Title says it all: it would be nice to have support for main Linux distributions packages repositories, Debian AFAIC.
This would provide a PPA like feature, just better :)

TY
J

**Comment by acabarbaye** [comment]
This would indeed be really helpful as currently a lot of projects only attach .deb or .apk or .rpm to releases, which defeats the initial purpose of easing the discovery, delivery, installation and setup of tools since standard package managers such as yum or apt cannot be used
